diffpy/snmf/subroutines.py

```diff
@@ -4,6 +4,64 @@
 import numdifftools
 
 
+def lift_data(data_input, lift=1):
+    """Lifts values of data_input
+
+    Adds 'lift' * the minimum value in data_input to data_input element-wise.
+
+    Parameters
+    ----------
+    data_input: 2d array like
+      The matrix containing a series of signals to be decomposed. Has dimensions N x M where N is the length of each
+      signal and M is the number of signals.
+
+    lift: float
+      The factor representing how much to lift 'data_input'.
+
+    Returns
+    -------
+    2d array like
+      The matrix that contains data_input - (min(data_input) * lift).
+
+    """
+    data_input = np.asarray(data_input)
+    return data_input - (np.min(data_input[:]) * lift)
```

Comment:
doesn't `data_input - (np.min(data_input) * lift)` work?  It is easier to read.

Reply:
It does. I will change it